content/topics/product-and-project-management/_index.md

```diff
@@ -10,6 +10,17 @@ title: "Product and project management"
 # description — keep it short and clear
 summary: ""
 
+aliases:
+  - /topics/agile/
+  - /topics/agile-lean-community-of-practice/
+  - /topics/kanban/
+  - /topics/lean/
+  - /topics/product-management/
+  - /topics/project-management/
```

Comment:
duplicate

Reply:
We need aliases for both product/project-management since they did exist as old topics.